ADOPTERS.md

```diff
@@ -43,6 +45,7 @@ If you would like to be included in this table, please submit a PR to this file
 * (Option) Description:
 
 ## Airspace Link
+
```

Comment:
should we delete the content here and only maintain `site/data/adopters.yaml` ?

Reply:
LGTM